designs/administrator/4.6/Pipelines/index.md

```diff
@@ -0,0 +1,72 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Pipelines
+
+## Description
+The Pipelines pages should be updated on the Admin console to be representative of a view that makes sense for the Admin user. Across all pages:
+ - The Namespace column appears when the All-projects selection has been made by the user.
+ - The kebab on the table functions as it does today.
+ - The Tech Preview badge has now been moved next to the title to make room for the Create button on the  right hand side.
+ - The Create button should be styled with the dropdown as it shows on the Secrets page.
+
+## Pipelines page
+On the Pipelines page the user can navigate between Pipelines, Pipeline Runs, Pipeline Resources, and Conditions via the tabs on the top of the page.
+- From the Create button the user can create Pipelines, Pipeline Resources, and Conditions.
+- The main action the user can take on the Pipelines page is creating a Pipeline from the Pipelines page
+- The Pipelines page contains the Pipelines Name, Last Run, Last Run Status, and Last Started columns
+- If the user selects all-projects from the project selector, the Namespace column appears.
+
+![Pipelines page](img/1-pipelines.jpg)
+
+## Pipeline Runs
+If the user selects all-projects from the project selector, the Namespace column appears. The Pipelines Runs page contains the Name, Status, Started, and Duration columns.
+
+![Pipeline runs](img/2-pipeline-runs.jpg)
+
+## Pipeline Resources
+If the user selects all-projects from the project selector, the Namespace column appears. The main action the user can take on the Pipelines Resources tab is creating a Pipeline Resource from the Pipeline page. The Pipelines Resources page contains the Name, Created, and Type columns.
+
+![Pipeline resources](img/3-pipeline-resources.jpg)
+
+## Condition
+If the user selects all-projects from the project selector, the Namespace column appears. The main action the user can take on the Condition tab is creating a Condition from the Pipelines page. The Pipelines Resources page contains the Name and Created columns
+
+![Condition](img/4-Condition.jpg)
+
+## Tasks
+If the user selects all-projects from the project selector, the Namespace column appears. The main action the user can take on the Tasks page is create a Task, Task Run, and Cluster Tasks.
+
+![Tasks](img/9-Tasks.jpg)
+
+## Task Runs
+If the user selects all-projects from the project selector, the Namespace column appears. The user can create a Task Run from the Task Run button.
+
+![Task Runs](img/10-Task-Runs.jpg)
```

Comment:
Today our Tasks Runs page doesn't show filters... so we're going to probably need to list the statuses out.

![image](https://user-images.githubusercontent.com/8126518/82940650-2506c200-9f63-11ea-9378-514a98d450fd.png)

@siamaksade correct me if I am wrong here, but those statuses _should_ follow the Pipeline statuses, no?

Reply:
This is referring to the Last Run status, correct?

Reply:
Each TaskRun has a status. It think we had it in the design doc if not mistaken.

@chmouel is Succeeded, Running, Failed and Cancelled the only TaskRun statuses?

Reply:
@rachael-phillips Sorry, I might have been confusing when I said Pipeline status. Technically it's the Pipeline Run status. Task Runs are instances of "runs" so they don't have "Last Run" as it is just the status of the Task Run. We could look at Tasks having a "Last Run" 🤔 Thoughts @siamaksade ? Not sure it's worth the effort personally, but I always go from a Pipeline down approach and not at the Task level all that often.

Reply:
here are all of task statuses https://github.com/tektoncd/pipeline/blob/cdae3707f/pkg/pod/status.go#L38

Not sure if i understood the full thread but let me know if you need me to review the whole thing...

Reply:
Interesting @chmouel - are failures not the same as Pipelines? It fails and then has a reason for the failure? Or are these all "states" where "failed" is something generic?

Reply:
@andrewballantyne and @chmouel any idea on how we should proceed here?

Reply:
This is not a new design @rachael-phillips let us just put a pin in finding out what the values are and just say there will be filters. No need to hold up designs around finding out this information.

Reply:
I see - thanks Andrew!

Reply:
@andrewballantyne @rachael-phillips for filters, could we group the statuses to "Running", "Success", and "Failed"? Failed is anything that is not Success or Running in this list:
https://github.com/tektoncd/pipeline/blob/cdae3707f/pkg/pod/status.go#L38

Reply:
That makes sense to me - what do you think @andrewballantyne?